src/Pages/People.tsx

```diff
@@ -0,0 +1,74 @@
+// import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
+import { Loader } from '../Components/Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { TablePeople } from '../Components/PeopleTable';
+
```

Comment:
Not all files was pushed
![2023-08-22_12h30_06](https://github.com/mate-academy/react_people-table-basics/assets/61904995/976332a4-3d82-4db3-a7e5-a4e5718d1785)